<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="number7_Button">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="number5_Button">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="number4_Button">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="clear_Button">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="number1_Button">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="dev_Button">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="numb6_Button">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="dot_Button">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="multi_Button">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="plus_Button">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="del_Button">
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="number3_Button">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="display_LineEdit"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="diff_Button">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="number2_Button">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="number9_Button">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="number8_Button">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="equval_Button">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="number0_Button">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="leftBracket_Button">
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="rightBracket_Button">
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
